netket/graph/lattice.py

```diff
@@ -19,6 +19,9 @@
 import itertools
 import networkx as _nx
 import warnings
+from typing import Tuple, Union
+import matplotlib
```

Comment:
we don't want to depend on matplotlib, I think you should import this only if draw is called

Reply:
no wait, did we just merge this import matplotlib when importing lattice?!? @PhilipVinc